2022-01-31 11:38:16,196 INFO    MainThread:24714 [internal.py:wandb_internal():87] W&B internal server running at pid: 24714, started at: 2022-01-31 11:38:16.195922
2022-01-31 11:38:16,196 INFO    MainThread:24658 [internal.py:wandb_internal():87] W&B internal server running at pid: 24658, started at: 2022-01-31 11:38:16.195387
2022-01-31 11:38:16,196 INFO    MainThread:24689 [internal.py:wandb_internal():87] W&B internal server running at pid: 24689, started at: 2022-01-31 11:38:16.195947
2022-01-31 11:38:16,196 INFO    MainThread:24642 [internal.py:wandb_internal():87] W&B internal server running at pid: 24642, started at: 2022-01-31 11:38:16.195818
2022-01-31 11:38:16,196 INFO    MainThread:24730 [internal.py:wandb_internal():87] W&B internal server running at pid: 24730, started at: 2022-01-31 11:38:16.195187
2022-01-31 11:38:16,196 INFO    MainThread:24622 [internal.py:wandb_internal():87] W&B internal server running at pid: 24622, started at: 2022-01-31 11:38:16.195829
2022-01-31 11:38:16,196 INFO    MainThread:24615 [internal.py:wandb_internal():87] W&B internal server running at pid: 24615, started at: 2022-01-31 11:38:16.195831
2022-01-31 11:38:16,196 INFO    MainThread:24697 [internal.py:wandb_internal():87] W&B internal server running at pid: 24697, started at: 2022-01-31 11:38:16.195194
2022-01-31 11:38:16,198 DEBUG   HandlerThread:24658 [handler.py:handle_request():130] handle_request: check_version
2022-01-31 11:38:16,198 DEBUG   HandlerThread:24689 [handler.py:handle_request():130] handle_request: check_version
2022-01-31 11:38:16,198 DEBUG   HandlerThread:24642 [handler.py:handle_request():130] handle_request: check_version
2022-01-31 11:38:16,198 INFO    WriterThread:24615 [datastore.py:open_for_write():77] open: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb
2022-01-31 11:38:16,198 INFO    WriterThread:24622 [datastore.py:open_for_write():77] open: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb
2022-01-31 11:38:16,198 DEBUG   HandlerThread:24714 [handler.py:handle_request():130] handle_request: check_version
2022-01-31 11:38:16,198 INFO    WriterThread:24658 [datastore.py:open_for_write():77] open: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb
2022-01-31 11:38:16,198 DEBUG   HandlerThread:24622 [handler.py:handle_request():130] handle_request: check_version
2022-01-31 11:38:16,198 DEBUG   HandlerThread:24730 [handler.py:handle_request():130] handle_request: check_version
2022-01-31 11:38:16,198 DEBUG   HandlerThread:24615 [handler.py:handle_request():130] handle_request: check_version
2022-01-31 11:38:16,198 INFO    WriterThread:24642 [datastore.py:open_for_write():77] open: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb
2022-01-31 11:38:16,198 INFO    WriterThread:24689 [datastore.py:open_for_write():77] open: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb
2022-01-31 11:38:16,198 INFO    WriterThread:24697 [datastore.py:open_for_write():77] open: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb
2022-01-31 11:38:16,198 INFO    WriterThread:24730 [datastore.py:open_for_write():77] open: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb
2022-01-31 11:38:16,199 DEBUG   HandlerThread:24697 [handler.py:handle_request():130] handle_request: check_version
2022-01-31 11:38:16,199 INFO    HandlerThread:24697 [handler.py:finish():731] shutting down handler
2022-01-31 11:38:16,198 INFO    WriterThread:24714 [datastore.py:open_for_write():77] open: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb
2022-01-31 11:38:16,201 INFO    SenderThread:24658 [sender.py:finish():1070] shutting down sender
2022-01-31 11:38:16,201 INFO    SenderThread:24689 [sender.py:finish():1070] shutting down sender
2022-01-31 11:38:16,201 INFO    SenderThread:24642 [sender.py:finish():1070] shutting down sender
2022-01-31 11:38:16,201 INFO    SenderThread:24714 [sender.py:finish():1070] shutting down sender
2022-01-31 11:38:16,202 DEBUG   SenderThread:24622 [sender.py:send():234] send: header
2022-01-31 11:38:16,202 INFO    SenderThread:24697 [sender.py:finish():1070] shutting down sender
2022-01-31 11:38:16,202 INFO    SenderThread:24615 [sender.py:finish():1070] shutting down sender
2022-01-31 11:38:16,202 DEBUG   SenderThread:24622 [sender.py:send_request():248] send_request: check_version
2022-01-31 11:38:16,203 INFO    SenderThread:24730 [sender.py:finish():1070] shutting down sender
2022-01-31 11:38:16,302 DEBUG   SenderThread:24622 [sender.py:send():234] send: run
2022-01-31 11:38:16,442 INFO    SenderThread:24622 [dir_watcher.py:__init__():169] watching files in: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files
2022-01-31 11:38:16,442 INFO    SenderThread:24622 [sender.py:_start_run_threads():804] run started: rwfihjum with start time 1643647094
2022-01-31 11:38:16,442 DEBUG   SenderThread:24622 [sender.py:send():234] send: summary
2022-01-31 11:38:16,443 DEBUG   HandlerThread:24622 [handler.py:handle_request():130] handle_request: run_start
2022-01-31 11:38:16,444 DEBUG   HandlerThread:24622 [meta.py:__init__():40] meta init
2022-01-31 11:38:16,444 INFO    SenderThread:24622 [sender.py:_save_file():939] saving file wandb-summary.json with policy end
2022-01-31 11:38:16,444 DEBUG   HandlerThread:24622 [meta.py:__init__():54] meta init done
2022-01-31 11:38:16,445 DEBUG   HandlerThread:24622 [meta.py:probe():214] probe
2022-01-31 11:38:16,447 DEBUG   HandlerThread:24622 [git.py:repo():33] git repository is invalid
2022-01-31 11:38:16,447 DEBUG   HandlerThread:24622 [meta.py:_save_pip():58] save pip
2022-01-31 11:38:16,448 DEBUG   HandlerThread:24622 [meta.py:_save_pip():72] save pip done
2022-01-31 11:38:16,448 DEBUG   HandlerThread:24622 [meta.py:_save_conda():79] save conda
2022-01-31 11:38:17,199 INFO    HandlerThread:24615 [handler.py:finish():731] shutting down handler
2022-01-31 11:38:17,199 INFO    HandlerThread:24689 [handler.py:finish():731] shutting down handler
2022-01-31 11:38:17,199 INFO    HandlerThread:24642 [handler.py:finish():731] shutting down handler
2022-01-31 11:38:17,199 INFO    HandlerThread:24658 [handler.py:finish():731] shutting down handler
2022-01-31 11:38:17,200 INFO    HandlerThread:24730 [handler.py:finish():731] shutting down handler
2022-01-31 11:38:17,200 INFO    HandlerThread:24714 [handler.py:finish():731] shutting down handler
2022-01-31 11:38:17,200 ERROR   MainThread:24658 [internal.py:wandb_internal():158] Thread WriterThread:
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 52, in run
    self._run()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    self._process(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal.py", line 332, in _process
    self._wm.write(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 30, in write
    self.open()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 26, in open
    self._ds.open_for_write(self._settings.sync_file)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/datastore.py", line 79, in open_for_write
    self._fp = open(fname, open_flags)
FileExistsError: [Errno 17] File exists: '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb'
2022-01-31 11:38:17,199 ERROR   MainThread:24697 [internal.py:wandb_internal():158] Thread WriterThread:
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 52, in run
    self._run()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    self._process(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal.py", line 332, in _process
    self._wm.write(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 30, in write
    self.open()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 26, in open
    self._ds.open_for_write(self._settings.sync_file)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/datastore.py", line 79, in open_for_write
    self._fp = open(fname, open_flags)
FileExistsError: [Errno 17] File exists: '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb'
2022-01-31 11:38:17,200 ERROR   MainThread:24642 [internal.py:wandb_internal():158] Thread WriterThread:
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 52, in run
    self._run()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    self._process(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal.py", line 332, in _process
    self._wm.write(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 30, in write
    self.open()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 26, in open
    self._ds.open_for_write(self._settings.sync_file)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/datastore.py", line 79, in open_for_write
    self._fp = open(fname, open_flags)
FileExistsError: [Errno 17] File exists: '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb'
2022-01-31 11:38:17,200 ERROR   MainThread:24730 [internal.py:wandb_internal():158] Thread WriterThread:
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 52, in run
    self._run()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    self._process(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal.py", line 332, in _process
    self._wm.write(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 30, in write
    self.open()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 26, in open
    self._ds.open_for_write(self._settings.sync_file)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/datastore.py", line 79, in open_for_write
    self._fp = open(fname, open_flags)
FileExistsError: [Errno 17] File exists: '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb'
2022-01-31 11:38:17,200 ERROR   MainThread:24714 [internal.py:wandb_internal():158] Thread WriterThread:
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 52, in run
    self._run()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    self._process(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal.py", line 332, in _process
    self._wm.write(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 30, in write
    self.open()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 26, in open
    self._ds.open_for_write(self._settings.sync_file)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/datastore.py", line 79, in open_for_write
    self._fp = open(fname, open_flags)
FileExistsError: [Errno 17] File exists: '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb'
2022-01-31 11:38:17,199 ERROR   MainThread:24689 [internal.py:wandb_internal():158] Thread WriterThread:
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 52, in run
    self._run()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    self._process(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal.py", line 332, in _process
    self._wm.write(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 30, in write
    self.open()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 26, in open
    self._ds.open_for_write(self._settings.sync_file)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/datastore.py", line 79, in open_for_write
    self._fp = open(fname, open_flags)
FileExistsError: [Errno 17] File exists: '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb'
2022-01-31 11:38:17,199 ERROR   MainThread:24615 [internal.py:wandb_internal():158] Thread WriterThread:
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 52, in run
    self._run()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    self._process(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/internal.py", line 332, in _process
    self._wm.write(record)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 30, in write
    self.open()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/writer.py", line 26, in open
    self._ds.open_for_write(self._settings.sync_file)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/internal/datastore.py", line 79, in open_for_write
    self._fp = open(fname, open_flags)
FileExistsError: [Errno 17] File exists: '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb'
2022-01-31 11:38:17,445 INFO    Thread-8  :24622 [dir_watcher.py:_on_file_created():217] file/dir created: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/requirements.txt
2022-01-31 11:38:17,445 INFO    Thread-8  :24622 [dir_watcher.py:_on_file_created():217] file/dir created: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/wandb-summary.json
2022-01-31 11:38:17,446 INFO    Thread-8  :24622 [dir_watcher.py:_on_file_created():217] file/dir created: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/conda-environment.yaml
2022-01-31 11:38:18,446 INFO    Thread-8  :24622 [dir_watcher.py:_on_file_modified():230] file/dir modified: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/conda-environment.yaml
2022-01-31 11:38:18,455 DEBUG   HandlerThread:24622 [meta.py:_save_conda():89] save conda done
2022-01-31 11:38:18,455 DEBUG   HandlerThread:24622 [meta.py:probe():252] probe done
2022-01-31 11:38:18,461 DEBUG   SenderThread:24622 [sender.py:send():234] send: files
2022-01-31 11:38:18,461 INFO    SenderThread:24622 [sender.py:_save_file():939] saving file wandb-metadata.json with policy now
2022-01-31 11:38:18,471 DEBUG   HandlerThread:24622 [handler.py:handle_request():130] handle_request: stop_status
2022-01-31 11:38:18,471 DEBUG   SenderThread:24622 [sender.py:send_request():248] send_request: stop_status
2022-01-31 11:38:18,593 DEBUG   SenderThread:24622 [sender.py:send():234] send: run
2022-01-31 11:38:18,740 INFO    SenderThread:24622 [sender.py:send_run():702] updated run: rwfihjum
2022-01-31 11:38:19,009 INFO    Thread-11 :24622 [upload_job.py:push():137] Uploaded file /tmp/tmpz8maa803wandb/13v0xj44-wandb-metadata.json
2022-01-31 11:38:19,213 INFO    MainThread:24697 [internal.py:handle_exit():77] Internal process exited
2022-01-31 11:38:19,214 INFO    MainThread:24658 [internal.py:handle_exit():77] Internal process exited
2022-01-31 11:38:19,214 INFO    MainThread:24615 [internal.py:handle_exit():77] Internal process exited
2022-01-31 11:38:19,214 INFO    MainThread:24642 [internal.py:handle_exit():77] Internal process exited
2022-01-31 11:38:19,214 INFO    MainThread:24730 [internal.py:handle_exit():77] Internal process exited
2022-01-31 11:38:19,214 INFO    MainThread:24714 [internal.py:handle_exit():77] Internal process exited
2022-01-31 11:38:19,214 INFO    MainThread:24689 [internal.py:handle_exit():77] Internal process exited
2022-01-31 11:38:19,452 INFO    Thread-8  :24622 [dir_watcher.py:_on_file_modified():230] file/dir modified: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/config.yaml
2022-01-31 11:38:19,452 INFO    Thread-8  :24622 [dir_watcher.py:_on_file_created():217] file/dir created: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/wandb-metadata.json
2022-01-31 11:38:19,452 INFO    Thread-8  :24622 [dir_watcher.py:_on_file_created():217] file/dir created: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/output.log
2022-01-31 11:38:33,594 DEBUG   HandlerThread:24622 [handler.py:handle_request():130] handle_request: stop_status
2022-01-31 11:38:33,595 DEBUG   SenderThread:24622 [sender.py:send_request():248] send_request: stop_status
2022-01-31 11:38:44,509 DEBUG   SenderThread:24622 [sender.py:send():234] send: stats
2022-01-31 11:38:48,713 DEBUG   HandlerThread:24622 [handler.py:handle_request():130] handle_request: stop_status
2022-01-31 11:38:48,713 DEBUG   SenderThread:24622 [sender.py:send_request():248] send_request: stop_status
2022-01-31 11:39:03,833 DEBUG   HandlerThread:24622 [handler.py:handle_request():130] handle_request: stop_status
2022-01-31 11:39:03,833 DEBUG   SenderThread:24622 [sender.py:send_request():248] send_request: stop_status
2022-01-31 11:39:14,574 DEBUG   SenderThread:24622 [sender.py:send():234] send: stats
2022-01-31 11:39:18,950 DEBUG   HandlerThread:24622 [handler.py:handle_request():130] handle_request: stop_status
2022-01-31 11:39:18,951 DEBUG   SenderThread:24622 [sender.py:send_request():248] send_request: stop_status
2022-01-31 11:39:34,078 DEBUG   HandlerThread:24622 [handler.py:handle_request():130] handle_request: stop_status
2022-01-31 11:39:34,079 DEBUG   SenderThread:24622 [sender.py:send_request():248] send_request: stop_status
2022-01-31 11:39:41,175 DEBUG   SenderThread:24622 [sender.py:send():234] send: history
2022-01-31 11:39:41,176 DEBUG   SenderThread:24622 [sender.py:send():234] send: summary
2022-01-31 11:39:41,178 INFO    SenderThread:24622 [sender.py:_save_file():939] saving file wandb-summary.json with policy end
2022-01-31 11:39:41,178 DEBUG   SenderThread:24622 [sender.py:send():234] send: history
2022-01-31 11:39:41,178 DEBUG   SenderThread:24622 [sender.py:send():234] send: summary
2022-01-31 11:39:41,179 INFO    SenderThread:24622 [sender.py:_save_file():939] saving file wandb-summary.json with policy end
2022-01-31 11:39:41,520 INFO    Thread-8  :24622 [dir_watcher.py:_on_file_modified():230] file/dir modified: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/wandb-summary.json
2022-01-31 11:39:44,639 DEBUG   SenderThread:24622 [sender.py:send():234] send: stats
2022-01-31 11:39:45,282 WARNING MainThread:24622 [internal.py:is_dead():366] Internal process exiting, parent pid 22821 disappeared
2022-01-31 11:39:45,282 ERROR   MainThread:24622 [internal.py:wandb_internal():142] Internal process shutdown.
2022-01-31 11:39:45,640 INFO    WriterThread:24622 [datastore.py:close():281] close: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/run-rwfihjum.wandb
2022-01-31 11:39:45,640 INFO    SenderThread:24622 [sender.py:finish():1070] shutting down sender
2022-01-31 11:39:45,640 INFO    SenderThread:24622 [dir_watcher.py:finish():283] shutting down directory watcher
2022-01-31 11:39:45,640 INFO    HandlerThread:24622 [handler.py:finish():731] shutting down handler
2022-01-31 11:39:46,522 INFO    SenderThread:24622 [dir_watcher.py:finish():313] scan: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files
2022-01-31 11:39:46,523 INFO    SenderThread:24622 [dir_watcher.py:finish():327] scan save: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/requirements.txt requirements.txt
2022-01-31 11:39:46,523 INFO    SenderThread:24622 [dir_watcher.py:finish():327] scan save: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/output.log output.log
2022-01-31 11:39:46,524 INFO    SenderThread:24622 [dir_watcher.py:finish():327] scan save: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/wandb-metadata.json wandb-metadata.json
2022-01-31 11:39:46,526 INFO    SenderThread:24622 [dir_watcher.py:finish():327] scan save: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/conda-environment.yaml conda-environment.yaml
2022-01-31 11:39:46,527 INFO    SenderThread:24622 [dir_watcher.py:finish():327] scan save: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/config.yaml config.yaml
2022-01-31 11:39:46,527 INFO    SenderThread:24622 [dir_watcher.py:finish():327] scan save: /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/wandb-summary.json wandb-summary.json
2022-01-31 11:39:46,530 INFO    SenderThread:24622 [file_pusher.py:finish():177] shutting down file pusher
2022-01-31 11:39:46,533 INFO    SenderThread:24622 [file_pusher.py:join():182] waiting for file pusher
2022-01-31 11:39:46,948 INFO    Thread-12 :24622 [upload_job.py:push():137] Uploaded file /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/requirements.txt
2022-01-31 11:39:46,983 INFO    Thread-15 :24622 [upload_job.py:push():137] Uploaded file /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/wandb-summary.json
2022-01-31 11:39:47,046 INFO    Thread-14 :24622 [upload_job.py:push():137] Uploaded file /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/config.yaml
2022-01-31 11:39:47,094 INFO    Thread-13 :24622 [upload_job.py:push():137] Uploaded file /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_113814-rwfihjum/files/conda-environment.yaml
2022-01-31 11:39:47,788 INFO    MainThread:24622 [internal.py:handle_exit():77] Internal process exited
